client/my-sites/backup/backup-date-picker/test/index.jsx

```diff
@@ -56,205 +60,161 @@ describe( 'BackupDatePicker', () => {
 
 	test( "Shows 'Yesterday' as the previous date if the current date is today", () => {
 		const today = moment();
-
-		const picker = shallow( <BackupDatePicker selectedDate={ today } onDateChange={ () => {} } /> );
-
-		const previousDate = picker.find(
-			'.backup-date-picker__select-date--previous .backup-date-picker__display-date'
+		const { asFragment } = render(
+			<BackupDatePicker selectedDate={ today } onDateChange={ () => {} } />
 		);
-		expect( previousDate.text() ).toEqual( 'Yesterday' );
+
+		expect( asFragment() ).toMatchSnapshot();
```

Comment:
This will be a problem, because calling `moment()` will return the date for today, but on the next day, it will be for tomorrow. This will cause a different snapshot and it will keep failing every day. Should we just use a static date here to address this problem?